CWE/events/algeria.txt

```diff
@@ -193,41 +216,34 @@ country_event = {
 
 	option = {
 		name = "EVT_8000319_A" # And that is precisely what we will do
-		government = theocracy
-		ruling_party_ideology = traditionalist
-		upper_house = {
-			ideology = traditionalist
-			value = 1
-		}
-		country_event = 800051
-		badboy = 25
-		add_country_modifier = { name = baby_boom duration = 7000 }
-		add_country_modifier = { name = contraception_decay duration = 17500 }
+		badboy = 15
 		any_country = {
 			limit = {
-				NOT = { government = theocracy }
-				OR = { neighbour = THIS is_greater_power = yes ai = no }
+				AND = {
+					NOT = { government = theocracy }
+					neighbour = THIS
+				}
 			}
 			casus_belli = { target = THIS type = liberate_country months = 60 }
 		}
-		random_country = { # ALG
-			limit = { has_country_flag = algerian_civil_war }
-			all_core = { add_core = ISI }
-			# Terrorism
-			random_owned = { province_event = 1511119 }
-			sphere_owner = { random_owned = { province_event = 1511119 } }
+		ALG = {
+			any_core = { add_core = ISI }
```

Comment:
Effect scope for this is all_core. Trigger scope is any_core

Reply:
Alright, sorry. I was trying to keep as much of that event's 'native' code as possible, but it was completely a mess. I think it might've been from nwo? I still barely have any idea what its supposed to do or when its supposed to fire. The person that reported that it was bugged had to manually fire it and its original trigger and effects are hilariously confusing to parse through. 

I assume its just supposed to be an event for the ISI tag to intervene in Algeria?

Reply:
Yeah it was from nwo and I’m afraid I don’t actually know what the event was for.

Reply:
It uses the same generic loc as the Algerian Civil War event and its title is 'Islamist Victory in the Algerian Civil War'. The Algerian Civil War lets you become the ISI tag, but then it wouldn't make sense to be at war with Algeria, but then at the same time the event to declare war turns you into a theocracy, purges an inane number of people, and assumes you were already in control of Algeria (while also having to be neighboring Algeria).

I agree that it might be better to remove it, and will do so now. I think for ISI, if the event is ever to be 'revived', could use a 'generic' system similar to the UAR with neighboring Arabic states - except those with traditionalist ruling parties rather than pan-Arabic ones.

Reply:
Alright